community/kernel/src/main/java/org/neo4j/kernel/impl/api/KernelStatement.java

```diff
@@ -170,7 +171,6 @@ void assertOpen()
     void initialize( StatementLocks statementLocks )
     {
         this.statementLocks = statementLocks;
-        this.timeoutMillis = clock.currentTimeMillis();
```

Comment:
Can we remove this? Will all statement timeouts in all transactions in the entire system have the statement timeout? KernelStatement and KTI instances are pooled and reused between transactions and so `initialize` method is basically their constructors.